packages/repl/src/lib/Output/srcdoc/index.html

```diff
@@ -151,7 +151,7 @@
 				window.addEventListener('unhandledrejection', (event) => {
 					parent.postMessage({ action: 'unhandledrejection', value: event.reason }, '*');
 				});
-			}.call(this));
+			}).call(this);
```

Comment:
I don't understand how the previous syntax worked, and am scared this will break.

Reply:
I'm just gonna trust prettier, it never fails me.